frontend/public/components/RBAC/bindings.jsx

```diff
@@ -266,14 +266,15 @@ class ListDropdown_ extends React.Component {
   render () {
     const {desc, fixed, placeholder, id, loaded} = this.props;
     const items = {};
+    const sortedItems = _.keys(this.state.items).sort();
```

Comment:
I Assume this will always be a "small" list and hence its fine to sort on the client? ideally it would be better to get it sorted  in advance?

Reply:
I don't believe it's possible through the kube API.

Reply:
I didn't expect that - thanks @spadgett

Reply:
@spadgett actually - https://kubernetes.io/docs/reference/kubectl/cheatsheet/#viewing-finding-resources implies its possible.

Reply:
afaik, kubectl also sorts on the client

Reply:
I tested kubectl with `--sort-by=.metadata.name --loglevel=10`, and it looks like there's no query parameter it sends for sorting

Reply:
@spadgett thanks for trying, you are right (https://github.com/kubernetes/kubernetes/blob/64bb688ecec332ee4e1ecdad1f3582fb61f82bae/pkg/kubectl/cmd/get/get.go#L363)